### PR TITLE
PLAT-519

### PR DESCRIPTION
**Pull request by tomParty**
Added the ability to include or not (default) test files for evaluation of a changed package. 

After approval, I will import this into the go repo as a tool, and remove the lib/tainted dir

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -120,14 +122,31 @@ func readPackages() {
...
+		cmdArgs = []string{
+			"--no-pager",
+			"-C",
+			*gitDirPtr,
+			"diff",
+			"--",
+			`":!*test*"`,
+			"--name-only",
+			*commitFromPtr,
+			*commitToPtr,
```
Comment (vandevusse) [inline main.go:148]: This could probably be refactored a little as both arrays are similar but it's not a major concern for me.

**Review by vandevusse — APPROVED** [review]

File: main.go [hunk 2]
```diff
@@ -120,14 +122,31 @@ func readPackages() {
...
+		}
+	default:
+		cmdArgs = []string{
+			"--no-pager",
+			"-C",
+			*gitDirPtr,
+			"diff",
+			"--",
+			`":!*test*"`,
```
Comment (pdt256) [inline main.go]: Test files in Go end with a `_test.go` suffix.

There may be files with names that contain **test**.
* con**test**ant
* la**test**
* **test**imonial
* etc.

**Review by pdt256 — CHANGES_REQUESTED** [review]
One tweak to update test file name pattern.

File: main.go (hunk 2 above)
Comment (tomParty) [reply main.go]: ahhh good catch.. i'll be more explicit

**Review by pdt256 — APPROVED** [review]

**Pull request merged**